src/jdk.management/unix/classes/com/sun/management/internal/OperatingSystemImpl.java

```diff
@@ -49,8 +49,8 @@ class OperatingSystemImpl extends BaseOperatingSystemImpl
     private ContainerCpuTicks processLoadTicks = new ProcessCpuTicks();
 
     private abstract class ContainerCpuTicks {
-        private long usageTicks = 0;
-        private long totalTicks = 0;
+        private volatile long usageTicks = 0;
+        private volatile long totalTicks = 0;
```

Comment:
You can drop initialising the fields to 0 (their default value) while you are at it as they are volatile writes.

Reply:
OK